### ISSUE TITLE
Jaws does not speak the labels of some fields when tabbing through dragon's add new command dialogue

### ISSUE THREAD

**Issue by pranavlal** [dictationbridge/dictationbridge-jfw#1]
Jaws does not read the correct labels of the fields when you tab through the "add new command" dialogue. To duplicate this, take the following steps.
#Launch Dragon.
#Navigate to the tools menu and select the option to add a new command.
#Tab through the dialogue box and note the labels spoken. Labels for the fields for the type of command to create and the state of the command are not spoken correctly. The values in the fields are spoken as opposed to the labels of the fields.

**Comment by ahicks92** [comment]
I'm going to see if we can't get Sean to look at the Jaws scripting side because he's going to be much, much faster at it than I am.
